Code/GraphMol/MolStandardize/Tautomer.cpp

```diff
@@ -202,6 +201,21 @@ std::vector<ROMOL_SPTR> TautomerEnumerator::enumerate(
   PRECONDITION(!modifiedBonds || modifiedBonds->size() >= mol.getNumBonds(),
                "bitset too small");
 
+  std::unique_ptr<boost::dynamic_bitset<>> matp;
+  if (!modifiedAtoms) {
+    // we need the modified atom for internal purposes. If the user didn't give
+    // us one, we have to make our own:
+    matp.reset(new boost::dynamic_bitset<>(mol.getNumAtoms()));
+    modifiedAtoms = matp.get();
+  }
+  std::unique_ptr<boost::dynamic_bitset<>> mbp;
+  if (!modifiedBonds) {
```

Comment:
This won't actually return the modified bonds to the caller if this is important.

Reply:
or atoms.

Reply:
It does say internal, so I'm guessing no...

Reply:
Yeah, the idea here is that the user hasn't indicated that they don't care (by either not providing `modifiedAtoms` or `modifiedBonds` or by providing them as `nullptr`). We need the data in this function, so we allocate memory for it here.